css/style.css

```diff
@@ -75,7 +76,282 @@ body {
+  font-weight: 700;
+  font-size: 17px;
+  color: #fff;
+}
+
+.aboutme {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.aboutme h1 {
+  margin: 120px 24px 0 24px;
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 72px;
+  color: #ff6b00;
+}
+
+.aboutme p {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  max-width: 327px;
+  margin: 24px;
+  color: #f4f5f7;
+}
+
+.aboutme .about-button {
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  background-color: #ff6b00;
+  width: 161px;
+  height: 48px;
+  text-align: center;
+  padding: 12px;
+  font-weight: 700;
+  margin: 24px;
+}
+
+.about-button a {
+  color: #fff;
+}
+
+.picture-background {
+  display: flex;
+  justify-content: flex-end;
+  max-width: 327px;
+}
+
+.end-line {
+  max-width: 280px;
+  margin: 12px 24px;
+  border-bottom: 1px solid #fff;
+}
+
+.languages {
+  max-width: 327px;
+}
+
+.languages div {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+  justify-content: center;
+  margin-top: 100px;
+}
+
+.form-background {
+  background-color: #fff;
+  max-width: 327px;
+}
+
+.formsection {
+  max-width: 327px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.formsection h3 {
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  margin: 100px 24px 24px;
+}
+
+.formsection input {
+  width: 280px;
+  margin: 12px;
+  border: none;
+  border-bottom: 1px solid #dbd8d7;
+  padding: 10px;
+}
+
+.formsection textarea {
+  max-width: 327px;
+  height: 114px;
+  margin: 24px;
+  background-color: #dbd8d7;
+  padding: 12px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+}
+
+.formsection .button {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  width: 131px;
+  height: 48px;
+  margin: 24px 24px 100px 24px;
+  background-color: #ff6b00;
+  text-align: center;
+  padding: 12px;
+}
+
+.formsection .button a {
+  color: #fff;
+}
+
+.form-background .icons {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+  justify-content: center;
+  padding: 20px;
+  margin-top: 1px;
+}
+
+.languages div h3 {
+  font-weight: 700;
+  font-size: 32px;
+  margin-top: -10px;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+}
+
+.workone .stack h3 {
+  font-weight: 500;
+  font-size: 15px;
+  font-family: 'Inter', sans-serif;
+  padding: 10px;
+  margin: 10px 10px 10px 0;
+  border: 1px solid #8993a4;
+}
+
+.worktwo .stacktwo h3 {
+  font-weight: 500;
+  font-size: 15px;
+  font-family: 'Inter', sans-serif;
+  padding: 10px 12px;
+  margin: 12px;
+  background: rgba(255, 255, 255, 0.24);
+  color: #fff;
+}
+
+.workone {
+  display: flex;
+  flex-direction: column;
+  max-width: 327px;
+  margin: 24px;
+}
+
+.worktwo {
+  height: 386px;
+  width: 327px;
+  background-image: url(../images/Img\ Placeholder.png);
+  margin: 12px;
+}
```

Comment:
- [ ] Impresive job @chibunduonyeje1, Also, According to the project requirement, every work section should be styled using a `display: flex` property. Please make sure to use `flexbox` to style all your work sections. This is to provide confidence that you can manipulate both flexbox and grid arrangements.
- [ ] Another thing is every work better be aligned to the center as provided on the Figma template. As you can see down 

### Figma design 
![button2](https://user-images.githubusercontent.com/69077061/205424300-06839952-be8c-45c9-96a3-c4c5844766d0.PNG)

### Your design 

![button](https://user-images.githubusercontent.com/69077061/205424299-c7d42bfd-6e29-459c-9a18-e8e6ee93d7d3.PNG)